_spinnaker/operator.md

```diff
@@ -259,89 +261,89 @@ kubectl -n <namespace> describe spinnakerservice spinnaker
 ```bash
 kubectl -n <namespace> delete spinnakerservice spinnaker
 ```
- 
+
 # Migrating from Halyard to Operator
 
-If you have a current Spinnaker instance installed via Halyard, use this guide to migrate existing 
+If you have a current Spinnaker instance installed via Halyard, use this guide to migrate existing
```

Comment:
Replace `via` w/ `with`

People at Armory have a bad habit of using things like `via` :(. It can trip up people whose first language isn't English.